2016-contributing-to-your-career.md

```diff
@@ -0,0 +1,189 @@
+
+Not only is open source a great way to build software, it's a great way to learn, exercise your skills, and showcase your experience. This talk will explore ways of contributing to open source–through code and beyond–that can help advance your career and open up new opportunities.
+
+## Intro
+
+Some of the most successful people I know in software development have arrived where they are because their participation in open source helped them learn, gain experience, build a reputation, and get a good job.
+
+So I asked them:
+
+### “How has open source impacted your career?”
+
+And I heard a _lot_ of responses like:
+
+> “ I would not be a software engineer today if it weren't for open source.”
+
+…and:
+
+> “I believe my career opportunities have come from my contributions to open source with my interests and skills being public.”
+
+…and…
+
+> “Open source has expanded my network in a way I could not have done on my own.”
+
+You hear people like Matthew Garret, a well-known Linux contributor, [saying](http://mjg59.dreamwidth.org/30420.html)
+
+> “Without free software I'd have started years later. I'd have lost the opportunity to collaborate with people spread over the entire world. My first job would have looked very different, as would my entire career since then. Without free software, almost everything I've achieved in my adult life would have been impossible.”
+
+Participating in open source will set you on path of continuously learning, building amazing things with amazing people, and finding great opportunities.
+
+Contents:
+
+1. [The value of contributing](#the-value-of-contributing)
+2. [Becoming a contributor](#becoming-a-contributor)
+3. [Building a reputation](#building-a-reputation)
+
+## The value of contributing
+
+Participating in open source has incredible advantages, like…
+
+- …working with and learning from some of the best developers in the world…
+- …on interesting technology that gets used by hundreds, thousands, or even millions of people…
+- …from wherever in the world you call "home"…
+- …with free tools that you love or want to learn to love…
+- …in communities with people that share your passions…
+- …all for free.
+
+Those alone are great reasons.  This talk takes those for granted, and instead focuses on something more selfish: your career.
+
+**What do Disney, Walmart, and John Deer have in common?** One is an "American diversified multinational mass media and entertainment conglomerate" ([wikiepdia](https://en.wikipedia.org/wiki/The_Walt_Disney_Company), another is the largest retailer in the world, and the third makes tractors.
```

Comment:
Also, I suspect you're talking about [John Deere](https://www.deere.com/en_US/regional_home.page), if you're talking about the US tractor manufacturer.